runware/base.py

```diff
@@ -439,6 +439,14 @@ def get_canny_object() -> Dict[str, int]:
                 request_object["CFGScale"] = requestImage.CFGScale
             if requestImage.seedImage:
                 request_object["seedImage"] = requestImage.seedImage
+            if requestImage.teaCache:
+                request_object["teaCache"] = requestImage.teaCache
+                request_object['teacache_distance'] = requestImage.teacache_distance
+            if requestImage.deepCache:
+                request_object["deepCache"] = requestImage.deepCache
+                request_object['deepcache_interval'] = requestImage.deepcache_interval
+                request_object['deepcache_branch_id'] = requestImage.deepcache_branch_id
+                request_object['deepcache_skip_mode'] = requestImage.deepcache_skip_mode
```

Comment:
this can just be `request_object["pipelineOptions"]` and assign the pipelineOptions object